website/docs/r/vdc_group.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_vdc_group"
+sidebar_current: "docs-vcd-resource-vdc-group"
+description: |-
+  Provides a VDC group resource.
+---
+
+# vcd\_vdc\_group
+Supported in provider *v3.5+* and VCD 10.2+.
+
+Provides a resource to manage VDC groups.
+
+~> Only `System Administrator` and `Org Users` with right `View VDC Group`, `Configure VDC Group`, `vDC Group: Configure Logging`, `Organization vDC Distributed Firewall: Enable/Disable` can manage VDC groups using this resource.
+
```

Comment:
Additional note that this resource supports only NSX-T VDC groups would be useful.